packages/compass-collection/src/components/collection-header-actions/view-information.tsx

```diff
@@ -1,10 +1,10 @@
-import { css } from '@mongodb-js/compass-components';
+import { css, spacing } from '@mongodb-js/compass-components';
 import React from 'react';
 
 const collectionHeaderActionsReadonlyStyles = css({
-  margin: '0px 8px',
+  margin: `0px ${spacing[2]}px`,
   marginLeft: 'auto',
-  fontSize: '16px',
+  fontSize: spacing[3],
```

Comment:
Is it allowed to use spacing for font sizes and line heights? I can't see this font size in leafygreen anywhere

Reply:
I don't know, I have seen it in other places in compass implemented like this, so I thought you have some kind of agreement. What should be used instead?

Reply:
I replaced all places in `compass-collection` where I used spacing for font sizes with some elements from typography. Pls let me know if this is ok I need to update it in some other way. I also removed line heights.